shuup/admin/static_src/base/less/shuup/buttons.less

```diff
@@ -0,0 +1,8 @@
+.btn {
+  border-radius: 0px;
+  border-width: 1px !important;
+  &.btn-red {
```

Comment:
I think it's wrong color, I was instructed to use color scheme defined for front page dashboard.